apps/explorer/src/app/components/txs/details/tx-stop-order-cancel.tsx

```diff
@@ -0,0 +1,63 @@
+import { t } from '@vegaprotocol/i18n';
+import type { BlockExplorerTransactionResult } from '../../../routes/types/block-explorer-response';
+import { MarketLink } from '../../links/';
+import type { TendermintBlocksResponse } from '../../../routes/blocks/tendermint-blocks-response';
+import { TxDetailsShared } from './shared/tx-details-shared';
+import { TableCell, TableRow, TableWithTbody } from '../../table';
+import DeterministicOrderDetails from '../../order-details/deterministic-order-details';
+import { CancelSummary } from '../../order-summary/order-cancellation';
+import Hash from '../../links/hash';
+
+interface TxDetailsStopOrderCancelProps {
+  txData: BlockExplorerTransactionResult | undefined;
+  pubKey: string | undefined;
+  blockData: TendermintBlocksResponse | undefined;
+}
+
+/**
+ * Someone cancelled a stop order
+ */
+export const TxDetailsStopOrderCancel = ({
+  txData,
+  pubKey,
+  blockData,
+}: TxDetailsStopOrderCancelProps) => {
+  if (!txData || !txData.command.stopOrdersCancellation) {
+    return <>{t('Awaiting Block Explorer transaction details')}</>;
+  }
+
+  const marketId: string = txData.command.stopOrdersCancellation.marketId;
+  const orderId: string = txData.command.stopOrdersCancellation.orderId;
```

Comment:
**thought (non-blocking)**: I suspect you can drop the `: string` when you switch to using the generated types